### PR TITLE
[SKIP CI] Cleanup README.md (#1791)

### LINKED ISSUE #1791: Cleanup README.md and fix broken links (if there are any)
We have pretty document and it is the time to cleanup/improve README.md by having a section with proper redirection. 

/CC @pdhamdhere @tusharnt

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
fixes #1791 

Cleaning up README.md and fixing broken links. With this PR our README will be pretty simple and directing links to our document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,42 +92,17 @@ logging config format for content details.
 
 **Docker**: 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)
 
-# Dependencies
+## Dependencies
 
-The plugin uses VMCI (Virtual Machine Communication Interface) and vSockets to contact the service on ESX. The associated Linux kernel drivers are installed via the VMware Tools and its open version, namely [open-vm-tools](https://github.com/vmware/open-vm-tools), packages. Either one of these packages must hence be installed in the guest OS. It's recommended to install the most uptodate version of either of these packages as available.
+The plugin uses VMCI (Virtual Machine Communication Interface) and vSockets to contact the service on ESX. The associated Linux kernel drivers are installed via the VMware Tools and its open version, namely [open-vm-tools](https://github.com/vmware/open-vm-tools), packages. Either one of these packages must be installed in the guest OS. It's recommended to install the most uptodate version of either of these packages as available.
```
Comment (ashahi1) [inline README.md]: uptodate >> up-to-date

File: README.md [hunk 2]
```diff
@@ -162,42 +92,17 @@ logging config format for content details.
...
--  Plugin fails to create volumes after installation on VM running boot2docker Linux. This is because open-vm-tools available for boot2docker doesn't install the vSockets driver and hence the plugin is unable to contact the ESX service. [#1744](https://github.com/vmware/docker-volume-vsphere/issues/1744)
--  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
+If you are running into any issue, please check out [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) related with vSphere docker volume service before raising a new issue.
 
 ## Contact us
+Please reach us out for any questions or problem you are facing with respective to vDVS. Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) to get contanct detail.
+
```
Comment (ashahi1) [inline README.md]: to get contanct detail.   >>> "contact"

File: README.md [hunk 3]
```diff
@@ -162,42 +92,17 @@ logging config format for content details.
...
--  Plugin fails to create volumes after installation on VM running boot2docker Linux. This is because open-vm-tools available for boot2docker doesn't install the vSockets driver and hence the plugin is unable to contact the ESX service. [#1744](https://github.com/vmware/docker-volume-vsphere/issues/1744)
--  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
+If you are running into any issue, please check out [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) related with vSphere docker volume service before raising a new issue.
 
 ## Contact us
+Please reach us out for any questions or problem you are facing with respective to vDVS. Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) to get contanct detail.
```
Comment (ashahi1) [inline README.md]: with respective to vDVS >>> with respect to vDVS

**Comment by shuklanirdesh82** [comment]
@ashahi1 Thanks! Your comments are addressed.

File: README.md [hunk 4]
```diff
@@ -29,89 +29,14 @@ The download consists of 2 parts:
...
-Pick the latest release and use the same version of vDVS plugin (on Docker host VM) and driver (on ESX).
-
-## Demos
-
-The demos are located on the project [site](https://vmware.github.io/docker-volume-vsphere/documentation) and [wiki](https://github.com/vmware/docker-volume-vsphere/wiki/Demos)
-
-## Project Website
-
-Project page is located @ [https://vmware.github.io/docker-volume-vsphere/](https://vmware.github.io/docker-volume-vsphere). Documentation, FAQ and other content can be found @ [https://vmware.github.io/docker-volume-vsphere/documentation](https://vmware.github.io/docker-volume-vsphere/documentation)
+Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) for vDVS driver's & plugin's installation instruction. Pick the latest release and use the same version of vDVS plugin (on Docker host VM) and driver (on ESX).
```
Comment (tusharnt) [inline README.md]: Please reword the text as follows - "for installation instructions for vDVS driver and managed plugin.
Download the release from <bintray location>. To ensure compatibility, make sure to use the same version of the vDVS driver and Managed plugin."

File: README.md [hunk 5]
```diff
@@ -173,46 +99,13 @@ logging config format for content details.
...
--  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
--  Admin CLI volume ls command is slow to list all the volumes if the setup has more than hundreds of volumes. [#1083](https://github.com/vmware/docker-volume-vsphere/issues/1083)
-
-## Known Differences Between Linux And Windows Plugin
-- Docker, by default, converts volume names to lower-case on Windows. Therefore, volume operations involving case-sensitive names will always be handled in lower case.
+If you are running into any issue, please check out [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) related with vSphere docker volume service before raising a new issue.
```
Comment (tusharnt) [inline README.md]: Rewording suggestion - Please check [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) to find out about known issues.

File: README.md [hunk 6]
```diff
@@ -173,46 +99,13 @@ logging config format for content details.
...
 
 ## Contact us
+Please reach us out for any questions or problem you are facing with respect to vDVS. Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) to get contact detail.
```
Comment (tusharnt) [inline README.md]: Let's just provide the link here - Contact us for file feature requests and report product issues! [http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html]

File: README.md [hunk 7]
```diff
@@ -173,46 +99,13 @@ logging config format for content details.
...
-    - 0.7 Release Overview
-        - [Some nice enhancements to Docker Volume Driver for vSphere v0.7](http://cormachogan.com/2016/10/06/nice-enhancements-docker-volume-driver-vsphere-v0-7/)
-- William Lam
-    - [Getting Started with Tech Preview of Docker Volume Driver for vSphere - updated](http://www.virtuallyghetto.com/2016/05/getting-started-with-tech-preview-of-docker-volume-driver-for-vsphere.html)
-- Virtual Blocks
-    - [vSphere Docker Volume Driver Brings Benefits of vSphere Storage to Containers](https://blogs.vmware.com/virtualblocks/2016/06/20/vsphere-docker-volume-driver-brings-benefits-of-vsphere-storage-to-containers/)
-    - [vSphere Docker Volume Service is now Docker Certified!](https://blogs.vmware.com/virtualblocks/2017/03/29/vsphere-docker-volume-service-now-docker-certified/)
-- Ryan Kelly
-    - [How to use the Docker Volume Driver for vSphere with Photon OS](http://www.vmtocloud.com/how-to-use-the-docker-volume-driver-for-vsphere-with-photon-os/)
+List of blogs related to vSphere Docker Volume Service are found at [vDVS Blogs](http://vmware.github.io/docker-volume-vsphere/documentation/blogs.html).
```
Comment (tusharnt) [inline README.md]: Let's just provide the link. No additional text is needed.

File: docs/misc/known-issues.md [hunk 8]
```diff
@@ -8,4 +8,7 @@ This section lists the major known issues with vSphere Docker Volume Service. Fo
...
 -  Plugin fails to create volumes after installation on VM running boot2docker Linux. This is because open-vm-tools available for boot2docker doesn't install the vSockets driver and hence the plugin is unable to contact the ESX service. [#1744](https://github.com/vmware/docker-volume-vsphere/issues/1744)
--  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
\ No newline at end of file
+-  Currently "vmdk-opsd stop" just stops (exits) the service forcefully. If there are operations in flight it could kill them in the middle of execution. This can potentially create inconsistencies in VM attachement, KV files or auth-db. [#1073](https://github.com/vmware/docker-volume-vsphere/issues/1073)
+
+## Known Differences Between Linux And Windows Plugin
+- Docker, by default, converts volume names to lower-case on Windows. Therefore, volume operations involving case-sensitive names will always be handled in lower case.
```
Comment (tusharnt) [inline docs/misc/known-issues.md]: Isn't this fixed now?

File: README.md [hunk 9]
```diff
@@ -148,13 +73,14 @@ journalctl -fu docker.service # Journalctl/Systemd
...
 
-**ESX Plugin logs**
+**vDVS Driver logs**
 
 * Log location: `/var/log/vmware/vmdk_ops.log`
 * Config file location: `/etc/vmware/vmdkops/log_config.json`  See Python
 logging config format for content details.
 * **Turning on debug logging**: replace all 'INFO' with 'DEBUG' in config file, restart the service
 
+Detailed information regarding vDVS configuration is found at [vDVS configuration page](http://vmware.github.io/docker-volume-vsphere/documentation/configuration.html).
```
Comment (tusharnt) [inline README.md]: Rewording suggestion : Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsphere/documentation/configuration.html) for detailed steps.

File: docs/misc/known-issues.md (hunk 8 above)
Comment (shuklanirdesh82) [reply docs/misc/known-issues.md]: The required issue is fixed and the summary is for https://github.com/vmware/docker-volume-vsphere/pull/1800#discussion_r135167886

File: README.md (hunk 9 above)
Comment (shuklanirdesh82) [reply README.md]: done!

File: README.md (hunk 7 above)
Comment (shuklanirdesh82) [reply README.md]: done!

File: README.md (hunk 6 above)
Comment (shuklanirdesh82) [reply README.md]: done!

File: README.md (hunk 5 above)
Comment (shuklanirdesh82) [reply README.md]: done!

File: README.md [hunk 10]
```diff
@@ -22,14 +22,14 @@ Detailed documentation can be found on our [GitHub Documentation Page](http://vm
...
-**[Click here to download from Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
+** Download releases from [Github releases](https://github.com/vmware/docker-volume-vsphere/releases) page**
 
 The download consists of 2 parts:
 
 1. VIB (vDVS driver): The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
 2. Managed plugin (vDVS plugin): Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
 
-Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) for vDVS driver's & plugin's installation instruction. Pick the latest release and use the same version of vDVS plugin (on Docker host VM) and driver (on ESX).
+Please check [vDVS Installation User Guide](http://vmware.github.io/docker-volume-vsphere/documentation/install.html) to get started. To ensure compatibility, make sure to use the same version of vDVS driver and managed plugin.
```
Comment (pshahzeb) [inline README.md]: Though appropriate, I suggest being specific here as follows 
vDVS managed plugin (on Docker host VM) and vDVS driver (on ESX).

This isn't completely intuitive to a person who wants to use this for the first time and is not at all familiar with infrastructure required.

Comment (shuklanirdesh82) [reply README.md]: agreed!!

**Pull request merged**